### PR TITLE
API-10382 - Implement auth doc updates

### PR DESCRIPTION
**Pull request by mheusgit**
### Description
Implements: API-10382 Implement auth doc updates for Clinical Health API

PR is three content changes located on the authorization page:
Add "launch" row with text as noted, including link to Scopes section
Add "launch" row with text as noted, including link to Requesting Authorization section
Add "fhirUser" row with text as noted, including link to fhirUserScope section

### CODE REVIEW

File: src/components/oauthDocs/AuthCodeFlowContent.tsx [hunk 1]
```diff
@@ -179,6 +179,37 @@ https://sandbox-api.va.gov${baseAuthPath}/authorization?
               </p>
             </td>
           </tr>
+          <tr>
+            <td>
+              <code>launch</code>
+            </td>
```
Comment (christineyost) [inline src/components/oauthDocs/AuthCodeFlowContent.tsx:185]: From Darcy's document it sounds like this row should only show when Clinical Health API is the selected

**Review by christineyost — COMMENTED** [review]
Looks good to me! The only thing is I think the launch parameter in the first table is only supposed to show if the Clinical Health API is selected

File: src/components/oauthDocs/AuthCodeFlowContent.tsx (hunk 1 above)
Comment (mheusgit) [reply src/components/oauthDocs/AuthCodeFlowContent.tsx]: Its been corrected

**Review by christineyost — APPROVED** [review]
Looks good to me!

**Pull request merged**